xbmc/games/GameUtils.cpp

```diff
@@ -252,3 +282,34 @@ bool CGameUtils::IsStandaloneGame(const ADDON::AddonPtr& addon)
 
   return false;
 }
+
+bool CGameUtils::Install(const std::string& gameClient)
+{
+  // If the addon isn't installed we need to install it
+  bool bInstalled = CServiceBroker::GetAddonMgr().IsAddonInstalled(gameClient);
```

Comment:
```suggestion
  bool installed = CServiceBroker::GetAddonMgr().IsAddonInstalled(gameClient);
```

no hungarian notation please